powerscale/helper/synciq_global_settings.go

```diff
@@ -63,7 +63,7 @@ func ManageSyncIQGlobalSettings(ctx context.Context, plan models.SyncIQGlobalSet
 	}
 
 	if toUpdate.SourceNetwork != nil {
-		if (toUpdate.SourceNetwork.Subnet == "" && toUpdate.SourceNetwork.Pool != "") || (toUpdate.SourceNetwork.Subnet != "" && toUpdate.SourceNetwork.Pool == "") {
+		if (toUpdate.SourceNetwork.Subnet == "" && toUpdate.SourceNetwork.Pool != "") || (toUpdate.SourceNetwork.Subnet != "" && toUpdate.SourceNetwork.Pool == "") || (toUpdate.SourceNetwork.Subnet == "" && toUpdate.SourceNetwork.Pool == "") {
```

Comment:
if toUpdate.SourceNetwork.Subnet == "" || toUpdate.SourceNetwork.Pool == "" {} will it work for all your case?

Reply:
Removed now